runtime/appruntime/cors/cors.go

```diff
@@ -23,12 +29,14 @@ func Options(cfg *config.CORS) cors.Options {
 	hasWildcardOriginWithoutCreds := cfg.AllowOriginsWithoutCredentials == nil || sortedSliceContains(originsWithoutCreds, "*")
 	hasUnsafeWildcardOriginWithCreds := sortedSliceContains(originsCreds, config.UnsafeAllOriginWithCredentials)
 
-	allowedHeaders := append([]string{"Authorization", "Content-Type"}, cfg.ExtraAllowedHeaders...)
+	allowedHeaders := append([]string{"Origin", "Authorization", "Content-Type"}, cfg.ExtraAllowedHeaders...)
```

Comment:
Is this needed? Seems surprising, how could CORS work without Origin headers?

Reply:
I'd rather fix the docs; it's surprising to have code which is misleading or doesn't have a purpose